Common/Source/Comm/device.cpp

```diff
@@ -550,6 +550,36 @@ static BOOL devInit(PDeviceDescriptor_t d){
     return(TRUE);
 }
 
+
+
+bool IsIdenticalPort(int i, int j)
+{
+  if (_tcscmp(szPort[i], _T("UDPServer")) == 0)
+  {
+    if (dwIpPort[i] == dwIpPort[j])
+      return true;
+  }
+  else
+  if (_tcscmp(szPort[i], _T("TCPServer")) == 0)
+  {
+    if (dwIpPort[i] == dwIpPort[j])
+      return true;
+  }
+  else
+  if (_tcscmp(szPort[i], _T("TCPClient")) == 0)
+  {
+    if (_tcscmp(szIpAddress[i], szIpAddress[j]) == 0)
+      if (dwIpPort[i] == dwIpPort[j])
+        return true;
```

Comment:
it's better to test "dwIpPort" before "szIpAddress"

Reply:
done:
https://github.com/LK8000/LK8000/pull/1328